app/assets/javascripts/components/profile_card_head.es6.jsx

```diff
@@ -0,0 +1,50 @@
+class ProfileCardHead extends React.Component {
+
+  static get propTypes() {
+    return {
+      cardName: React.PropTypes.string.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      cardName: '',
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        position: 'relative',
+        height: 40,
+        verticalAlign: 'middle',
+        width: '100%',
+      },
+      cardName: {
+        position: 'relative',
+        float: 'left',
+        lineHeight: '40px',
+        marginLeft: 5,
+        fontSize: 24,
```

Comment:
`fontSize` needs `'px'` not by conventions, but because without it the browser would not know what units to use.
`lineHeight